CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 ### Jsonnet
 
 * [CHANGE] Dashboards: Remove per-user series legends from Tenants dashboard. #1605
+* [CHANGE] Dashboards: Show in-memory series and the per-user series limit on Tenants dashboard. #1613
```

Comment:
I think dashboard changes should go in the `### Mixin` section, not `### Jsonnet` which is the jsonnet for k8s

Reply:
Good spot, I'll fix it for the last commit too.